pages/_app.tsx

```diff
@@ -19,17 +43,27 @@ export default class MyApp extends App {
 
 	render(): JSX.Element {
 		const { Component, pageProps } = this.props
+		const theme = MakeTheme(this.state.themeType)
 
 		return (
 			<>
 				<Head>
 					{/* https://err.sh/next.js/no-document-title */}
-					<title>My new cool app</title>
+					<title>Impact</title>
```

Comment:
Can you add `<meta name="theme-color" content={theme.palette.primary.main} />` below the title here?